modules/cli/src/test/kotlin/datamaintain/cli/AppTest.kt

```diff
@@ -437,6 +438,25 @@ internal class AppTest {
                                 .isFalse()
                     }
                 }
+
+                @Nested
+                inner class MongoSh {
+                    @Test
+                    fun `should build config with mongosh`() {
+                        // Given
+                        val argv = updateMongoDbMinimumArguments().plus("--mongo-shell").plus("mongosh")
+
+                        // When
+                        runApp(argv)
+
+                        // Then
+                        expectThat((configWrapper.datamaintainConfig!!.driverConfig) as MongoDriverConfig)
+                            .and {
+                                get { mongoShell }.isEqualTo(MongoShell.MONGOSH)
+                                get { clientPath }.isEqualTo(Paths.get("mongosh"))
+                            }
+                    }
+                }
             }
```

Comment:
In my opinion the following test is missing:
- check that the selected shell is mongo when on calling CLI with ```--mongo-shell mongo```